resources/views/waitingrooms/partials/_form_waitingrooms.blade.php

```diff
@@ -2,7 +2,7 @@
 
 <div class="form-item -padded">
     {{ Form::label('contest_id', 'Contest ID:', ['class' => 'field-label']) }}
-    {{ Form::text('contest_id', NULL, ['class' => 'text-field']) }}
+    {{ Form::text('contest_id', NULL, ['class' => 'text-field', 'readonly' => 'readonly']) }}
```

Comment:
I wasn't 100% sure how blade would handle the attribute if no value is added (needs to be passed in to the input as an associative array). Technically the HTML5 properties like `disabled`, `readonly`, etc, which can exist as just stand alone attributes without the `=`, also work as `disabled = "disabled"` and `readonly = "readonly"` so it doesn't hurt it; just a bit redundant.